pkg/diff/chunk.go

```diff
@@ -34,69 +35,67 @@ var (
 	gt    = ">"
 	gte   = ">="
 
-	bucketMode = "bucket"
+	bucketMode = "bucketMode"
 	normalMode = "normalMode"
 )
 
-type bound struct {
-	column      string
-	lower       string
-	lowerSymbol string
-	upper       string
-	upperSymbol string
+// Bound represents a bound for a column
+type Bound struct {
+	Column      string `json:"column"`
+	Lower       string `json:"lower"`
+	LowerSymbol string `json:"lower-symbol"`
+	Upper       string `json:"upper"`
+	UpperSymbol string `json:"upper-symbol"`
 }
 
-// chunkRange represents chunk range
-type chunkRange struct {
-	bounds []*bound
+// ChunkRange represents chunk range
+type ChunkRange struct {
```

Comment:
I think we should only create object as [POC](https://docs.google.com/document/d/1i43s9LBQEdIUkrdnAnZMa-69RhiG5VrJ8xcu3qWVRYE/edit?usp=sharing), there are two advantages
* Fit design document
* less code, we don't need code to assemble or transform the objects